python/ql/lib/semmle/python/frameworks/RestFramework.qll

```diff
@@ -194,6 +194,9 @@ private module RestFramework {
       exists(RestFrameworkApiViewClass vc |
         this.getParameter() =
           vc.getARequestHandler().(PrivateDjango::DjangoRouteHandler).getRequestParam()
+        or
+        // retrieve(self, request, **kwargs)
+        this.getParameter() = vc.getARequestHandler().(PrivateDjango::DjangoRouteHandler).getKwarg()
```

Comment:
This is the wrong place for this new modeling, since this QL class is about the `request` parameter, and as I understand you want to ensure that view-sets properly have routed-parameters + that they can flow to the `**kwargs` parameter?

Reply:
actually, let me get back to this PR in a little while, when I'm done with my local edits :+1: